.github/workflows/grandpa-cw-docker-image.yml

```diff
@@ -0,0 +1,74 @@
+name: "Build and publish Grandpa Docker image"
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  
+  build-and-publish:
+
+    name: Build & push docker image
+    
+    runs-on:
+      - self-hosted
+      - x64-monster
```

Comment:
By probably don't need to use self-hosted runner here. 

Lets try to use Github runners first:

```
runs-on: ubuntu-latest
```